onnxruntime/core/graph/contrib_ops/internal_nhwc_onnx_opset.cc

```diff
@@ -84,6 +100,8 @@ void OpSet_Internal_NHWC_ONNX::ForEachSchema(const std::function<void(ONNX_NAMES
   REGISTER_NHWC_SCHEMA(fn, QLinearConv, 10);
   REGISTER_NHWC_SCHEMA_FROM_MSDOMAIN(fn, QLinearAveragePool, 1);
   REGISTER_NHWC_SCHEMA_FROM_MSDOMAIN(fn, QLinearConvTranspose, 1);
+  RegisterNHWCSchemaWithSumActivation(
+      fn, contrib::GetOpSchema<contrib::ONNX_OPERATOR_SET_SCHEMA_CLASS_NAME(Microsoft, 1, FusedConv)>());
```

Comment:
The existing operators here are 1:1 with ONNX operators and their opsets, with the only difference being the conversion of NCHW inputs/outputs to NHWC. This relationship is leveraged by layout transformation as we can convert the ONNX layout sensitive operator to the NHWC version in the internal domain. FusedConv doesn't seem to quite fit that if it's not mirroring an ONNX operator. 

If we want to start using the kMSInternalNHWCDomain domain for NHWC contrib ops as well we'd need to make sure that that doesn't invalidate any assumptions made in other code (e.g. graph partitioning, transpose optimizer) about the ops in that domain. Mixing in ONNX and contrib ops potentially makes the domain a little more confusing though.

There are some existing contrib ops that support NHWC layout which are used by nhwc_transformer.cc/h. They have a 'channels_last' attribute. Would it be more consistent to take that approach so the existing NCHW FusedConv and new NCHW FusedConv simply differ by the usage of that attribute? That way they're in the same domain and like all contrib ops have a fixed opset of 1.